js/app/file-list-page/file-list-page/index.js

```
@@ -71,24 +72,32 @@ export class FileListPage extends Component {
    * @private
    */
   _initNestedComponentsWithDataFromServer() {
-    this._apiService.userPreferences()
-        .then((userPreferences) => {
-          this._userDetails.username = userPreferences.name;
+    this._stateService.dispatch(actionNames.getUserDetails).then(() => {
+      this._stateService.dispatch(actionNames.loadCurrentFolder).catch(() => {
+      });
```

Why do we need empty catch?

---

Delete.